test/bin/compiler-beta.t/run.t

```diff
@@ -0,0 +1,132 @@
+We have a simple project that only depends on OCaml
+
+  $ cat test.opam
+  opam-version: "2.0"
+  depends: [
+    "ocaml"
+    "dune"
+  ]
+  x-opam-monorepo-opam-repositories: [
+    "file://$OPAM_MONOREPO_CWD/minimal-repo"
+    "file://$OPAM_MONOREPO_CWD/repo-with-beta-ocaml"
+  ]
+
+We use the minimal repository which includes OCaml 4.13.1
+
+  $ gen-minimal-repo
+
+We also use an extra repository which happens to contain the packages of the
+ongoing beta release of OCaml 4.14.1
+
+  $ cat repo-with-beta-ocaml/packages/ocaml/ocaml.4.14.0/opam
```

Comment:
Similar as above. This adds 80 lines of which possibly only like 71 is important.

Reply:
You're right that printing those files entirely does not bring much value here. I'll remove that in favor of something shorter while still documenting those were taken from upstream opam repo.